ui-speedspacechart/src/components/SpeedSpaceChart.tsx

```diff
@@ -145,10 +148,11 @@ const SpeedSpaceChart = ({
           />
         </div>
       )}
+      <DeclivityLayer width={WIDTH_OFFSET} height={HEIGHT_OFFSET} store={store} />
       <CurveLayer width={WIDTH_OFFSET} height={HEIGHT_OFFSET} store={store} />
-      <AxisLayerY width={width} height={height} store={store} />
-      <MajorGridY width={width} height={height} store={store} />
-      <AxisLayerX width={width} height={height} store={store} />
+      <AxisLayerY width={declivities ? width - 42 : width} height={height} store={store} />
```

Comment:
This `declivities ? width - 42 : width` is too specific and custom to be repeated three times like this. The `42` should probably be a constant somewhere with a proper name, and the `declivities ? width - 42 : width` expression itself should be stored in a variable that describes what it represents.

Reply:
Done :)